src/build/entrypoint.ts

```diff
@@ -40,7 +40,18 @@ export async function generateEntrypoint(project: Project, opts: BuildOpts) {
 		// Import Prisma serverless adapter for Neon
 		import * as neon from "https://esm.sh/@neondatabase/serverless@^0.9.3";
 		import { PrismaNeon } from "https://esm.sh/@prisma/adapter-neon@^5.15.0";
-    // neon.neonConfig.coalesceWrites = false;
+		`;
+	} else if (opts.dbDriver == DbDriver.CloudflareHyperdrive) {
+		imports += `
+		// Import Prisma adapter for Postgres
+    //
+    // We can't use esm.sh for these because they rely on special Node
+    // functionality & don't need to be portable
+    //
+    // https://github.com/esm-dev/esm.sh/issues/684
+    // import 'npm:pg-cloudflare';
+		import pg from "npm:pg@8.12.0";
+		import { PrismaPg } from "npm:@prisma/adapter-pg@5.12.0";
```

Comment:
iirc we had a ticket for removing node compatibility from deployed workers, is that canceled now that we need to use node for hyperdrive? or is it handled by esbuild somehow?